macOS/DuckDuckGo/Bookmarks/Model/BookmarkManager.swift

```diff
@@ -203,10 +203,10 @@ final class LocalBookmarkManager: BookmarkManager {
     }
 
     @discardableResult func makeBookmark(for url: URL, title: String, isFavorite: Bool) -> Bookmark? {
-        makeBookmark(for: url, title: title, isFavorite: isFavorite, index: nil, parent: nil)
+        makeBookmark(for: url, title: title, isFavorite: isFavorite, index: nil, parent: nil, completion: { _ in })
```

Comment:
❓ Do we need to add the empty completion handler here? We already define the default argument in the makeBookmark function.

Reply:
Good catch, it’s not needed :) I’ve removed it.